src/assets/data/update.json

```diff
@@ -1 +1 @@
-{"updated":0}
+{"updated":1542351642822}
```

Comment:
remove this file from commit, this is generated automatically on build.

Reply:
Done https://github.com/deniszholob/factorio-cheat-sheet/pull/82/commits/52feba3e3e04c82afb201e7495c79cf7db08520f